base.css

```diff
@@ -0,0 +1,47 @@
+.nav{
```

Comment:
You may want to play around with this some more, but your ``nav`` should have some kind ``display: none`` or something of that sort so that it is hidden in mobile view and replaced with a hamburger menu per the instructions.

```
The nav menu should be hidden by default in mobile mode and should display when the user clicks the hamburger icon.
```